--- libdc1394-2.0.0/examples/Makefile.am.orig	2008-01-04 09:21:53.000000000 +0100
+++ libdc1394-2.0.0/examples/Makefile.am	2008-01-07 21:13:23.295114640 +0100
@@ -58,5 +58,5 @@
 if XVOK
 dc1394_multiview_CFLAGS = $(X_CFLAGS) $(XV_CFLAGS)
 dc1394_multiview_SOURCES = dc1394_multiview.c
-dc1394_multiview_LDADD = $(LDADD) ../dc1394/libdc1394.la $(X_LIBS) $(X_PRE_LIBS) $(XV_LIBS) -lX11 $(X_EXTRA_LIBS)
+dc1394_multiview_LDADD = $(LDADD) ../dc1394/libdc1394.la $(X_LIBS) $(XV_LIBS) -lX11 $(X_EXTRA_LIBS)
 endif
