diff -aurN libdc1394-1.1.0.orig/examples/grab_partial_image.c libdc1394-1.1.0/examples/grab_partial_image.c
--- libdc1394-1.1.0.orig/examples/grab_partial_image.c	2004-11-22 07:05:14.000000000 +0100
+++ libdc1394-1.1.0/examples/grab_partial_image.c	2006-02-12 19:54:17.888838888 +0100
@@ -213,7 +213,7 @@
     /*---------------------------------------------------------------------
      *  output elapsed time
      *---------------------------------------------------------------------*/
-    elapsed_time = (float)(times(&tms_buf) - start_time) / CLK_TCK;
+    elapsed_time = (float)(times(&tms_buf) - start_time) / CLOCKS_PER_SEC;
     printf( "got frame %d. elapsed time: %g sec ==> %g frames/second\n",
             i, elapsed_time, (float)i / elapsed_time);
   }
