--- libdc1394-2.1.2/examples/Makefile.am.orig	2009-06-11 06:05:47.000000000 +0200
+++ libdc1394-2.1.2/examples/Makefile.am	2010-06-23 22:50:51.331445612 +0200
@@ -48,4 +48,4 @@
 
 dc1394_multiview_CFLAGS = $(X_CFLAGS) $(XV_CFLAGS)
 dc1394_multiview_SOURCES = dc1394_multiview.c
-dc1394_multiview_LDADD = $(LDADD) $(X_LIBS) $(X_PRE_LIBS) $(XV_LIBS) -lX11 $(X_EXTRA_LIBS)
+dc1394_multiview_LDADD = $(LDADD) $(X_LIBS) $(XV_LIBS) -lX11 $(X_EXTRA_LIBS)
