--- libdc1394-2.0.2/acinclude.m4.orig	2008-05-01 07:14:50.000000000 +0200
+++ libdc1394-2.0.2/acinclude.m4	2008-07-12 00:31:26.522038059 +0200
@@ -78,7 +78,7 @@
 AC_PATH_TOOL([$1], [$2])
 if test "$DX_FLAG_[]DX_CURRENT_FEATURE$$1" = 1; then
     AC_MSG_WARN([$2 not found - will not DX_CURRENT_DESCRIPTION])
-    AC_SUBST([DX_FLAG_[]DX_CURRENT_FEATURE], 0)
+    AC_SUBST([DX_FLAG_][DX_CURRENT_FEATURE], 0)
 fi
 ])
 
@@ -101,7 +101,7 @@
 # ----------------------------------------------------------
 # Turn off the DX_CURRENT_FEATURE if the required feature is off.
 AC_DEFUN([DX_CLEAR_DEPEND], [
-test "$DX_FLAG_$1" = "$2" || AC_SUBST([DX_FLAG_[]DX_CURRENT_FEATURE], 0)
+test "$DX_FLAG_$1" = "$2" || AC_SUBST([DX_FLAG_][DX_CURRENT_FEATURE], 0)
 ])
 
 # DX_FEATURE_ARG(FEATURE, DESCRIPTION,
--- libdc1394-2.0.2/configure.in.orig	2008-05-08 03:15:15.000000000 +0200
+++ libdc1394-2.0.2/configure.in	2008-07-12 01:01:23.882034707 +0200
@@ -3,11 +3,7 @@
 AC_INIT([libdc1394],[2.0.2],[libdc1394-devel@lists.sf.net])
 AC_CONFIG_SRCDIR([dc1394/control.c])
 
-case "$host" in
-*-*-cygwin* | *-*-mingw*)
 AC_LIBTOOL_WIN32_DLL
-;;
-esac
 
 #
 DX_HTML_FEATURE(OFF)
