--- libdc1394-2.2.1/examples/Makefile.am~	2013-01-28 04:43:18.000000000 +0200
+++ libdc1394-2.2.1/examples/Makefile.am	2015-03-03 18:19:41.324454128 +0200
@@ -66,7 +66,7 @@
 
 dc1394_multiview_CFLAGS = $(X_CFLAGS) $(XV_CFLAGS)
 dc1394_multiview_SOURCES = dc1394_multiview.c
-dc1394_multiview_LDADD = $(LDADD) $(X_LIBS) $(X_PRE_LIBS) $(XV_LIBS) -lX11 $(X_EXTRA_LIBS)
+dc1394_multiview_LDADD = $(LDADD) $(X_LIBS) $(XV_LIBS) -lX11 $(X_EXTRA_LIBS)
 
 dc1394_iso_CFLAGS = $(SDL_CFLAGS)
 dc1394_iso_SOURCES = dc1394_iso.c
